inference_scripts/lp_infer/lp_infer_gnn.py

```diff
@@ -64,8 +64,15 @@ def main(args):
             'Supported test negative samplers include '
             f'[{BUILTIN_LP_UNIFORM_NEG_SAMPLER}, {BUILTIN_LP_JOINT_NEG_SAMPLER}]')
 
+    # Increase eval batch size to maximize reuse of node embeddings, and also increase
+    # network and GPU efficiency
+    avail_cuda_mem = th.cuda.get_device_properties(0).total_memory - th.cuda.memory_reserved(0)
+    req_cuda_mem_per_edge = config.num_negative_edges_eval * config._n_hidden * 4 * 2
```

Comment:
Can you explain why we choose *4*2?

Reply:
4 is the number of bytes in a float. 2 was for 2 node embeddings for each edge. Anyway, we are not using this solution anymore.